modules/monitoring-disabling-the-local-alertmanager.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+[id="disabling_the_local_alertmanager_{context}"]
+= Disabling the local Alertmanager
+
+A default installation of the {product-title} monitoring stack includes in the `openshift-monitoring` project an Alertmanager instance that routes alerts from Prometheus instances.
```

Comment:
change to below is better
`Alertmanager is a default installation of the {product-title} monitoring stack includes in the `openshift-monitoring` project  that routes alerts from Prometheus instances.`

Reply:
What about this?  

"A local Alertmanager instance that routes alerts from Prometheus instances is enabled by default in the `openshift-monitoring` project of the (product-title} monitoring stack."